sites/waterworld/server/templates/website-section/global-thought-leaders.marko

```diff
@@ -0,0 +1,129 @@
+import { getAsObject } from '@base-cms/object-path';
+import getAdUnit from '@endeavorb2b/base-website-common/utils/gam/get-adunit';
+import { video, whitepaper, company } from '../../api/fragments/global-thought-leaders';
+
+$ const { site } = out.global;
+$ const page = {
+  title: 'Global Thought Leader Series',
+  description: 'Certain companies in the water and wastewater industry are veritable trailblazers, forging ahead and sharing their expertise with others in an effort to help them stay current with new trends and technologies and do their jobs more efficiently. Our new Global Thought Leader Series will feature a collection of white papers and related videos from these innovators, giving you free and easy access to a bank of useful knowledge.',
+};
+$ const imageOpts = {
+  fit: 'clip',
+  h: 100,
+  minH: 100,
```

Comment:
Are you sure you only want 100 pixels?

Reply:
Pretty sure...I want them to be the same height so they layout the same throughout the page.